### PR TITLE
refactor: 98 pt pages with AI descriptions and keywords

### PR DESCRIPTION
**Pull request by gabriel-azion**

### CODE REVIEW

File: src/content/docs/pt-br/pages/arquiteturas/edge-application/serverless-apps.mdx [hunk 1]
```diff
@@ -1,9 +1,13 @@
 ---
 title: Implemente uma arquitetura serverless com a Azion
-description: Utilize a plataforma da Azion para criar aplicações serverless. Substitua as CDNs legadas por conteúdo edge-native para melhorar a velocidade, confiabilidade e eficiência de custo.
-meta_tags: edge, aplicações, aplicações web, api, entrega de conteúdo, cdn, serverless, Azion
-namespace: docs_arch_serverless_apps
+description: Descubra como utilizar a Edge Application da Azion para implementações
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/edge-application/serverless-apps.mdx:3]:
```suggestion
description: Descubra como utilizar o Edge Application da Azion para implementações
```

File: src/content/docs/pt-br/pages/arquiteturas/security-modernization/security-modernization-com-edge-computing.mdx [hunk 2]
```diff
@@ -1,9 +1,14 @@
 ---
 title: Acelere a modernização da segurança com edge computing
-description: Os recursos de edge computing podem lhe ajudar proteger suas aplicações e seu conteúdo, beneficiando-se de uma lógica moderna e produtos de observabilidade.
-meta_tags: arquitetura, segurança, edge firewall
-namespace: docs_arch_security_modernization_with_edge
+description: Explore como o Edge Computing e o Edge Firewall melhoram a segurança
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/security-modernization/security-modernization-com-edge-computing.mdx:3]:
```suggestion
description: Explore como a Edge Computing e o Edge Firewall melhoram a segurança
```

File: src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/devtools/tools-cli.mdx [hunk 3]
```diff
@@ -1,9 +1,13 @@
 ---
 title: Desenvolva com a Azion CLI
-description: A Azion CLI é uma ferramenta de código aberto que possibilita o gerenciamento dos serviços e recursos na plataforma da Azion.
-meta_tags: CLI
-namespace: documentation_dev_with_azion_cli
+description: Explore a Azion CLI para gerenciar aplicativos na Plataforma de Edge
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/devtools/tools-cli.mdx:3]:
```suggestion
description: Explore a Azion CLI para gerenciar aplicações na Plataforma de Edge
```

File: src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/devtools/tools-go.mdx [hunk 4]
```diff
@@ -1,10 +1,13 @@
 ---
 title: Construa com o Azion Go SDK
-description: O SDK da Azion para Go é um conjunto de ferramentas para desenvolvedores que simplifica o processo de interação com os serviços de API da Azion ao desenvolver aplicações na linguagem de programação Go.
-meta_tags: SDK, Go, Golang, development
+description: Explore o SDK Azion para Go, ferramental que facilita a interação com
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/devtools/tools-go.mdx:3]:
```suggestion
description: Explore o SDK Azion para Go, ferramenta que facilita a interação com
```

File: src/content/docs/pt-br/pages/deploy-jornada/configurar-node/provisionar-arquivos/provisionar-arquivos.mdx [hunk 5]
```diff
@@ -1,10 +1,14 @@
 ---
 title: Como provisionar arquivos
-description: Para orquestrar seus serviços nos seus edge nodes, primeiro, você precisa configurar todos os arquivos que serão provisionados.
-meta_tags: deploy, orchestrator
+description: Aprenda como configurar e gerenciar resources no Azion Console e Real-Time
+  Manager para otimizar seus Edge Services e impulsionar a eficiência do Edge Computing.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/deploy-jornada/configurar-node/provisionar-arquivos/provisionar-arquivos.mdx:4]:
```suggestion
  Manager para otimizar seus Edge Services e impulsionar a eficiência da Edge Computing.
```

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/create/create.mdx [hunk 6]
```diff
@@ -1,10 +1,14 @@
 ---
 title: Azion CLI create
-description: Crie edge applications, edge functions e outros recursos através da Azion CLI.
-meta_tags: cli, edge, create, development
+description: Descubra como criar Edge Applications, Edge Functions, e mais com Azion
+  CLI. Inclui configurações de cache, CDN, e storage avançado.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/create/create.mdx:4]:
```suggestion
  CLI. Inclui configurações de cache, CDN e storage avançado.
```

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/list/list.mdx [hunk 7]
```diff
@@ -1,10 +1,14 @@
 ---
 title: Azion CLI list
-description: Consulte suas edge applications, edge functions e outros recursos na plataforma através da Azion CLI.
-meta_tags: cli, edge, list, development
+description: Explore and manage Edge applications, functions, rules, and more using
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/list/list.mdx:3]: English

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/sync/sync.mdx [hunk 8]
```diff
@@ -1,10 +1,15 @@
 ---
 title: Azion CLI sync
-description: Sincronize seu arquivo local contendo a configuração dos recursos de sua aplicação existente com os recursos remotos através da Azion CLI.
-meta_tags: cli, next, javascript, astro, vue, react, edge computing
+description: Gerencie e sincronize configurações de aplicativos edge usando Azion
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/sync/sync.mdx:3]:
```suggestion
description: Gerencie e sincronize configurações de edge applications usando Azion
```

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/unlink/unlink.mdx [hunk 9]
```diff
@@ -1,10 +1,14 @@
 ---
 title: Azion CLI unlink
-description: Desvinculando um projeto existente da Plataforma de Edge Computing da Azion.
-meta_tags: cli, next, javascript, astro, vue, react, computação de borda
+description: Aprenda como usar o comando 'azion unlink' para desvincular projetos
+  da plataforma de Edge Computing Azion, incluindo opções como --yes e --help.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/unlink/unlink.mdx:4]:
```suggestion
  da plataforma de Edge Computing da Azion, incluindo opções como --yes e --help.
```

Comment (MarianaAguilera) [reply src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/unlink/unlink.mdx:4]: --yes, is it ok?

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/update/update.mdx [hunk 10]
```diff
@@ -1,10 +1,14 @@
 ---
 title: Azion CLI update
-description: Atualize suas edge applications, edge functions e outros recursos através da Azion CLI.
-meta_tags: cli, edge, update, development
+description: Aprenda a atualizar aplicações e funções edge, regras de engine, configurações
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/update/update.mdx:3]:
```suggestion
description: Aprenda a atualizar aplicações e funções edge, rules engine, configurações
```

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/other/edge-functions/edge-functions.mdx [hunk 11]
```diff
@@ -1,10 +1,14 @@
 ---
 title: Azion CLI edge-function
-description: Crie, liste, descreva, atualize e exclua suas edge functions na plataforma da Azion através da Azion CLI.
-meta_tags: cli, functions, javascript, serverless, devops
+description: 'Gerencie edge functions na plataforma Azion com a Azion CLI: criar,
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli/other/edge-functions/edge-functions.mdx:3]:
```suggestion
description: Gerencie edge functions na plataforma Azion com a Azion CLI: criar,
```

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/other/edge-functions/edge-functions.mdx [hunk 12]
```diff
@@ -1,10 +1,14 @@
...
+  listar, descrever, atualizar e excluir funções de forma eficiente.'
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli/other/edge-functions/edge-functions.mdx:4]:
```suggestion
  listar, descrever, atualizar e excluir funções de forma eficiente.
```

File: src/content/docs/pt-br/pages/devtools/console-kit/console-kit.mdx [hunk 13]
```diff
@@ -1,10 +1,14 @@
 ---
 title: Console Kit
-description: Desenvolvido com Vue/Vite, PrimeVue e Tailwind, o Azion Console Kit é um kit de desenvolvimento front-end usado para criar uma interface personalizada do Console Azion.
-meta_tags: edge, console, desenvolvimento, interface, vue, primevue
+description: 'Descubra o Azion Console Kit: um ambiente de desenvolvimento front-end
+  com Vue/Vite e Tailwind, ideal para personalizar e integrar interfaces no Edge Computing.'
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/console-kit/console-kit.mdx:4]:
```suggestion
description: Descubra o Azion Console Kit: um ambiente de desenvolvimento front-end
  com Vue/Vite e Tailwind, ideal para personalizar e integrar interfaces de Edge Computing.
```

File: src/content/docs/pt-br/pages/guias/acessar-azion-console/acessar-azion-console.mdx [hunk 14]
```diff
@@ -1,9 +1,16 @@
 ---
 title: Como acessar o Azion Console
-description: Esta documentação ajudará você a acessar a interface da Azion e explorar os produtos e recursos.
-meta_tags: Azion, edge application, journey, onboarding, edge computing
-namespace: docs_guide_access_console
+description: Explora a plataforma Azion e suas funcionalidades como Edge Application,
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/acessar-azion-console/acessar-azion-console.mdx:3]:
```suggestion
description: Explora a plataforma da Azion e suas funcionalidades como Edge Application,
```

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/list/list.mdx (hunk 7 above)
Comment (gabriel-azion) [reply src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/list/list.mdx:3]:
```suggestion
description: Consulte suas edge applications, edge functions e outros recursos na plataforma através da Azion CLI.
```

Comment (gabriel-azion) [reply src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/list/list.mdx:3]: Some things are stranger than stranger things

**Pull request merged**